### ISSUE TITLE
querystring stuff

### ISSUE THREAD

**Issue by jonathanong** [pillarjs/discussions#5]
i think @dougwilson and i discussed a querystring parser/stringifier that is more configurable and strict. specifically, i only ever want strings. haven't looked at the issue right now.

**Comment by dougwilson** [comment]
+1000; basically a way to get predictable types back

**Comment by Raynos** [comment]
:+1: We use `querystring` build into node core and then use our JSON schema definition of the HTTP request so that we can get predictable types.

We return a validation error from the querystring parser if it doesnt match the schema.

**Comment by rlidwka** [comment]
yeah, maybe just add a validation/transformation step and use existing querystring module?

**Comment by dougwilson** [comment]
We could at first, but it would make it slightly slower :)

**Comment by Fishrock123** [comment]
:+1: from all the people who don't need all the extra `qs` stuff. (fast would be nice though...)
